store/tikv/region_request.go

```diff
@@ -226,7 +227,8 @@ func (s *RegionRequestSender) SendReqCtx(
 	if span := opentracing.SpanFromContext(bo.ctx); span != nil && span.Tracer() != nil {
 		span1 := span.Tracer().StartSpan("regionRequest.SendReqCtx", opentracing.ChildOf(span.Context()))
 		defer span1.Finish()
-		bo = bo.Clone()
+		// TODO: Make sure trace works without cloning the backoffer.
```

Comment:
I prefer this way, otherwise, we don't know who is responsible for the TODO.
```suggestion
		// TODO(MyonKeminta): Make sure trace works without cloning the backoffer.
```

Reply:
Sounds good